homeassistant/components/unifiprotect/__init__.py

```diff
@@ -101,6 +102,20 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, data_service.async_stop)
     )
 
+    protect_version = data_service.api.bootstrap.nvr.version
+    if protect_version.is_prerelease:
+        ir.async_create_issue(
+            hass,
+            DOMAIN,
+            f"ea_warning_{protect_version}",
+            is_fixable=False,
+            is_persistent=False,
+            learn_more_url="https://www.home-assistant.io/integrations/unifiprotect#about-unifi-early-access",
+            severity=IssueSeverity.WARNING,
+            translation_key="ea_warning",
+            translation_placeholders={"version": str(protect_version)},
+        )
+
```

Comment:
```suggestion
        )
    else:
        async_delete_issue(hass, DOMAIN, f"ea_warning")

```

Reply:
If they uninstall early access we should make the warning go away (which I think its the desired outcome).

Reply:
The issue is not persistent, so it would simply just not be created again.